### PR TITLE
gleam 0.30 changes

### PR DESCRIPTION
**Pull request by inoas**
copied from https://github.com/gleam-lang/website/blob/4a3103783199c6e6b1b6c4b14c837c1ae7c71f85/book/javascript/highlightjs-gleam.js

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,18 +1,12 @@
...
-    "as assert case const external fn if import let " +
-    "panic opaque pub todo tuple type use";
+    "as assert case const fn if import let panic use opaque pub todo type";
   const STRING = {
     className: "string",
     variants: [{ begin: /"/, end: /"/ }],
     contains: [hljs.BACKSLASH_ESCAPE],
     relevance: 0,
   };
-  const NAME = {
```
Comment (inoas) [inline index.js:11]: What was this for? Should it be gone or does it miss in the website version of this file?

Comment (inoas) [reply index.js:11]: okay, I will use the website version but not remove the NAME part here and update the website version with the name part.

File: index.js [hunk 2]
```diff
@@ -80,10 +79,22 @@ module.exports = function (hljs) {
...
       {
         className: "keyword",
         beginKeywords: KEYWORDS,
       },
+      {
+        // Attributes
+        className: "title",
+        begin: "\\b@[a-z][a-za-z0-9_]*\\b",
+        relevance: 0,
+      },
```
Comment (lpil) [inline index.js:97]: Oh dear! This was a mistake on my part. This shouldn't be here, I've removed it from the website.

Comment (inoas) [reply index.js]: I hope it is correct now!

**Review by lpil — APPROVED** [review]
Thanks!!

**Pull request merged**